cmd/system/system.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// system contains a suite of Sponsored Apps for arkade
+package system
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func MakeSystem() *cobra.Command {
+
+	command := &cobra.Command{
+		Use:     "system",
+		Short:   "System apps",
+		Long:    `Apps for systems.`,
+		Aliases: []string{"v"},
```

Comment:
Why `v`?

Reply:
I copied from the venafi app so we can change that to `s`